src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -0,0 +1,285 @@
+            <TagsTable tags={event.tags} />
+          </SidebarBlock>
+        </SidebarColumn>
+      </ColumnGrid>
+    );
+  }
+
+  renderActiveTab(event, activeTab) {
+    const entry = event.entries.find(item => item.type === activeTab);
+    const [projectId, _] = this.props.eventSlug.split(':');
+    if (INTERFACES[activeTab]) {
+      const Component = INTERFACES[activeTab];
+      return (
+        <Component
+          projectId={projectId}
+          event={event}
+          type={entry.type}
+          data={entry.data}
+          isShare={false}
+        />
+      );
+    } else if (OTHER_SECTIONS[activeTab]) {
+      const Component = OTHER_SECTIONS[activeTab];
+      return <Component event={event} isShare={false} />;
+    } else {
+      /*eslint no-console:0*/
+      window.console &&
+        console.error &&
+        console.error('Unregistered interface: ' + entry.type);
+
+      return (
+        <EventDataSection event={event} type={entry.type} title={entry.type}>
+          <p>{t('There was an error rendering this data.')}</p>
+        </EventDataSection>
+      );
+    }
+  }
+
+  render() {
+    return (
+      <ModalContainer>
+        <CloseButton onClick={this.handleClose} size="zero" icon="icon-close" />
+        {this.renderBody()}
+      </ModalContainer>
+    );
+  }
+}
+
+const EventHeader = props => {
+  const {title} = getTitle(props.event);
+  return (
+    <div>
+      <h2>{title}</h2>
+      <p>{getMessage(props.event)}</p>
+    </div>
+  );
+};
+EventHeader.propTypes = {
+  event: SentryTypes.Event.isRequired,
+};
+
+const EventMetadata = props => {
+  const jsonUrl = 'TODO build this';
+  const {event} = props;
+
+  return (
+    <SidebarBlock withSeparator>
+      <MetadataContainer>ID {event.eventID}</MetadataContainer>
+      <MetadataContainer>
+        <DateTime
+          date={getDynamicText({value: event.dateCreated, fixed: 'Dummy timestamp'})}
+        />
+        <ExternalLink href={jsonUrl} className="json-link">
+          JSON (<FileSize bytes={event.size} />)
+        </ExternalLink>
+      </MetadataContainer>
+    </SidebarBlock>
+  );
+};
+EventMetadata.propTypes = {
+  event: SentryTypes.Event.isRequired,
+};
+
+const MetadataContainer = styled('div')`
+  display: flex;
+  justify-content: space-between;
+
+  color: ${p => p.theme.gray3};
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
+const ColumnGrid = styled('div')`
+  display: grid;
+  grid-template-columns: 70% 1fr;
+  grid-template-rows: auto;
+  grid-column-gap: ${space(3)};
+`;
+
+const ContentColumn = styled('div')`
+  grid-column: 1 / 2;
+`;
+
+const SidebarColumn = styled('div')`
+  grid-column: 2 / 3;
+`;
+
+const SidebarBlock = styled('div')`
+  margin: 0 0 ${space(2)} 0;
+  padding: 0 0 ${space(2)} 0;
+  ${p => (p.withSeparator ? `border-bottom: 1px solid ${p.theme.borderLight};` : '')}
+`;
+
+const ModalContainer = styled('div')`
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  right: 0px;
```

Comment:
Would we want to make this fit the window and have it internally scroll inside of the modal?

Reply:
I'm not sure what the desired behavior is right now. My thinking was that I could get this merged and then we could figure out how scrolling works. I see a few options.

1. Expand the modal to fill the viewport with a 20px gutter, freeze scrolling on the body and scroll the contents of the modal.
2. Size the modal based on its content and allow the page & modal to scroll as one unit.
3. Something else?

Reply:
I like #1 fwiw, I find page scrolling with the modal to be a bit awkward since the modal is kind of on top of the scroll bar